ci/run_premerge_cpu.sh

```
@@ -1,164 +1,164 @@
-        echo "Run all tests..."
-        verify_bundle
-        ;;
-    changed)
-        echo "Run changed tests..."
-        verify_bundle
-        ;;
-    *)
-        echo "ERROR: unknown parameter: $BUILD_TYPE"
-        ;;
-esac
+#!/bin/bash
```

Shellcheck (0.11.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)